src/components/UserDetailSections/UserAffiliations/UserAffiliations.js

```diff
@@ -46,10 +78,47 @@ const UserAffiliations = ({
 
   const isLoading = isFetching || isAffiliationsMutating;
 
-  const onUpdateAffiliations = useCallback(({ added, removed }) => {
-    return handleAssignment({ added, removed })
-      .then(refetch);
-  }, [handleAssignment, refetch]);
+  const onUpdateAffiliations = useCallback(async ({ added, removed }) => {
+    try {
+      const responses = await handleAssignment({ added, removed });
+      const errors = await getResponseErrors(responses);
```

Comment:
it should be done in useUserAffiliationsMutation

Reply:
@alisher-epam this comment is important

Reply:
@NikitaSedyx Yes, I am improving it.